src/main/java/taa/command/assessment/EditAssessmentCommand.java

```diff
@@ -65,7 +65,8 @@ public void checkArgument() throws TaaException {
             if (!Util.isStringDouble(newMaximumMarksString)) {
```

Comment:
Change to isStringDouble(maximumMarksString, 2) to only allow 2dp

Reply:
ok updated